File: DeliveryServiceBackend/DeliveryService/Model/Product.cs
using System.ComponentModel.DataAnnotations;

namespace DeliveryService.Model
{
  public class Product
  {
    public Product(string name, double price)
    {
      Name = name;
      Price = price;
    }

    public Product()
    {
      Name = "";
      Price = 0.0;
    }

    public int Id { get; set; }                     //Product ID
    public string Name { get; set; } = String.Empty;    //Product Name
    public double Price { get; set; }                   //Produict Price
  }
}

File: src/app/models/product.ts

export class Product
{
    Name         :string ;
    Price        :number ;
    Ingredients  :Array<string> ;

    constructor(name:string, price:number, ingredients  :Array<string>)
    {
      this.Name        = name;    
      this.Price       = price; 
      this.Ingredients = ingredients;     
    };
}